CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable, unreleased changes to this project will be documented in this file.
 ## [Unreleased]
 
 - Add our implementation of UUID scalar - #5646 by @koradon
+- Move tests inside tested modules - #5651 by @fowczarek
```

Comment:
Speaking of the changelog, I'm not sure all changes should be mentioned there, we currently treat it like a dumping ground for all commits that make it to master. I think we should start asking ourselves: is this information useful for users/developers?

Reply:
Agree, internal changes are not always worth mentioning. New features, bugfixes, changes in API shape or behavior are. I usually refine it a bit anyway before copying to release notes.